session_1_Harish_Gumnur/.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-3.0.0
```

Comment:
Hey, can you use the ruby 2.7.2 instead? The automated tests will not work otherwise.